corehq/apps/hqwebapp/two_factor_gateways.py

```diff
@@ -112,14 +112,17 @@ def get_ip_address():
 
     ip_address = get_ip_address()
     username = device.user.username
+    number = device.number
```

Comment:
Can this number be used "as-is"?

Reply:
you mean as opposed to being normalized first? I think it's fine to use as-is

Reply:
yeah wasn't sure if it needed to be formatted.